config_src/coupled_driver/ocean_model_MOM.F90

```diff
@@ -568,8 +576,9 @@ subroutine add_berg_flux_to_shelf(G, fluxes, use_ice_shelf, density_ice, kv_ice,
     call pass_vector(fluxes%frac_shelf_u, fluxes%frac_shelf_v, G%domain, TO_ALL, CGRID_NE)
     
     !Zero'ing out other fluxes under the tabular icebergs 
-    do j=jsd,jed ; do i=isd,ied
-        if (fluxes%frac_shelf_h(i,j) > 0.5) then  !Only applying for ice shelf covering most of cell
+    if (berg_area_threshold > 0.) then
```

Comment:
Shouldn't this ">0" be ">=0" so that if we can zero fluxes for frac_shelf_h=epsilon with berg_area_threshold=0?